src/components/Books.svelte

```diff
@@ -32,11 +33,14 @@
 </script>
 
 <ul>
+	<h2 class="font-Josefin text-amber-100 text-4xl font-normal text-center">Our Bookshelf</h2>
 	{#each books as book}
 		<li>
-			<h2>{book.title}</h2>
-			<a href={book.url}>{book.fileName}</a> <!--relative links are broken, placeholder for now-->
-			<img src={book.artworkPath} alt={book.title}>
+			<section class="bg-amber-100 max-w-xs m-auto py-5 mb-5 mt-5 text-center border-8 border-solid border-[#b9f6ca] rounded" >
```

Comment:
Why are you using a section inside each list item, rather than the list item itself?